frontend/src/pages/stakingOverview/Chains.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Chain } from './Chain'
+
+export const Chains = (props) => {
```

Comment:
This compoment is not required. Move list rendering to parent component.

Reply:
done